### PR TITLE
esmodules: Update lib/viewport

### PR DESCRIPTION
**Pull request by dmsnell**
@see: https://github.com/Automattic/wp-calypso/milestone/224

Previously we were exporting a default object of multiple methods and
importing those through the non-spec-compliant Babel destructuring
import statements.

This patch makes those methods proper named exports in the work of
turning off CommonJS compilation.

### CODE REVIEW

File: client/my-sites/domains/index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import domainsController from './controller';
 import domainManagementController from './domain-management/controller';
 import SiftScience from 'lib/siftscience';
 import config from 'config';
-import paths from './paths';
+import * as paths from './paths';
```
Comment (ockham) [inline client/my-sites/domains/index.js]: This looks like it doesn't belong to this PR, no?

Comment (dmsnell) [reply client/my-sites/domains/index.js]: thanks! I removed it in 16a792b
